authaad.properties

```diff
@@ -2,12 +2,18 @@ name=Azure Active Directory Authentication for SonarQube
 category=Integration
 description=Allows the use of Azure Active Directory as an authentication source for SonarQube.
 homepageUrl=https://github.com/hkamel/sonar-auth-aad
-publicVersions=1.1.0
-archivedVersions=1.0
+publicVersions=1.2.0
+archivedVersions=1.0,1.1.0
 
 defaults.mavenGroupId=org.almrangers.auth.aad
 defaults.mavenArtifactId=sonar-auth-aad-plugin
 
+1.2.0.description=Group sync enhancements including transitive members. Code cleanup.
+1.2.0.sqVersions=[7.0, LATEST]
```

Comment:
SQ versions <7.9 have been archived. Please set the beginning of this range to 7.9.

Reply:
Made the requested changes in a new commit in the PR.